framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -214,15 +214,6 @@ struct BPL_WLAN_PARAMS {
 
     /* Wi-Fi Channel (0 for ACS) */
     int channel;
-
-    /* Wi-Fi SSID */
-    char ssid[BPL_SSID_LEN];
-
-    /* Wi-Fi KeyPassphrase */
-    char passphrase[BPL_PASS_LEN];
-
-    /* Wi-Fi Securirt Mode */
-    char security[BPL_SEC_LEN];
```

Comment:
this information isn't used at all - although read periodically.
it is checked for change and the agent is alerted, but the agent ignores this information and only handles a change in `band_enabled`.

there is a task to remove the rest of the settings read from wireless config from there and read using NL message or similar.

Reply:
Reading this information in a different way is fine, but still it feels like this change should not be part of this PR.

Reply:
the reason to remove was that we are not using the code.
the reason why we were looking and this at all was that due to the difference in the wireless configuration between UGW, GLInet, RAX40, etc.

we could have changed those wifi settings to optional - but that would just hide the fact we aren't really reading it.
we could have invested in fixing the wireless configuration in GLInet and Turris Omnia - but the decision was not to invest in this at this point for several reasons:
1. the data read is not really used by the prplmesh
2. there is a task to remove the prplmesh access to wireless configuration and instead use BWL or some other method to retrieve the information.

regarding the commit message, I don't think that the platform wireless configuration difference is really beneficial information here since if we wanted to address it we would have fixed it.